Shapes.java

```diff
@@ -1,25 +1,23 @@
-import java.util.*;
+import java.until.*;
 
-public class Shapes{
-  public static void main(String[] args) {
-    Scanner input = new Scanner(System.in);
-    System.out.print("Enter the side length: ");
-    double side = input.nextDouble();
-    square(side);
-  }
+ public class Shapes{
+   public static void main(String[] args){
+     square();
+     triangle();
 
-  /*
-   * N: square
-   * P: to calculate the area of any square
-   * I: a side measure (double)
-   * R:
-   */
-  public static void square(double s){
-    //side times side
-    double a = s * s;
+     public static void square(){
```

Comment:
I like how you're using static methods here! Can you figure out a way to change the return type to something other than `void` in order to make the code more modular?